sovtokenfees/sovtokenfees/test/view_change/test_xfer_fees_during_view_change.py

```diff
@@ -0,0 +1,22 @@
+from sovtokenfees.test.view_change.helper import scenario_txns_during_view_change
+
+from sovtoken.constants import XFER_PUBLIC
+
+
+def fees():
```

Comment:
Do we miss `@pytest.fixture(scope="module")` here?

Reply:
Yes, thank you